UnityProject/Assets/Scripts/Items/Weapons/Projectiles/Behaviours/ProjectileIncendiary.cs

```diff
@@ -65,15 +65,9 @@ public bool OnHit(MatrixManager.CustomPhysicsHit hit)
 
 			if (hit.CollisionHit.GameObject == null) return true;
 
-			//TODO REMOVE AFTER MOBS ARE MOVED TO NEW HEALTH
-			if (hit.CollisionHit.GameObject.TryGetComponent(out LivingHealthMasterBase health))
```

Comment:
have mobs been moved over to new health?

Reply:
not yet, but I've been working on making damage work between all mobs again. not just the player.

Reply:
this was checking for the new class again a few lines below but yeah I guess I'll add it back with the old class